roles/cluster_logging/files/kibana/provision-index.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/sh
+
+#
+# Script to setup the default index pattern in kibana
+#
+
+INDEX_ID="logstash-*"
+
+KIBANA="http://kibana:443"
```

Comment:
We should variable this. Suggestion to use an environment variable, that will be injected by another key of the configmap

Reply:
These are exported as variables already:
```shell
$ kubectl run -ti shell --image=busybox --namespace=kube-ops --restart=Never
If you don't see a command prompt, try pressing enter.
/ # env | grep KIBANA
KIBANA_PORT_443_TCP=tcp://10.233.41.254:443
KIBANA_SERVICE_HOST=10.233.41.254
KIBANA_PORT=tcp://10.233.41.254:443
KIBANA_SERVICE_PORT=443
KIBANA_PORT_443_TCP_ADDR=10.233.41.254
KIBANA_PORT_443_TCP_PORT=443
KIBANA_PORT_443_TCP_PROTO=tcp
```

Reply:
Yes but those are environment variable, not config (what if mainstream or us change 'kibana' to 'kibana-ui' or 'kibana-api')